### PR TITLE
Add ansible playbooks to configure the prometheus servers and its components

### PR DESCRIPTION
**Pull request by Miyurz**
This PR is intended to provision the monitoring stack. Once terraform spins up the cluster, ansible is kicked in to provision the telemetry cluster and its components.

### CODE REVIEW

File: inventory [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[prometheus-servers]
+10.0.1.145
+10.0.1.225
+10.0.1.240
```
Comment (RahulMahale) [inline inventory]: @Miyurz Can we avoid IPs hard-coding ? Instead we can fetch it using cloud provider API or tags.

File: prometheus-servers.retry [hunk 2]
```diff
@@ -0,0 +1,3 @@
+10.0.1.145
+10.0.1.225
+10.0.1.240
```
Comment (RahulMahale) [inline prometheus-servers.retry]: I think we dont need .retry file in repository.

Comment (Miyurz) [reply prometheus-servers.retry]: looks like need to get these into .gitignore

File: inventory (hunk 1 above)
Comment (Miyurz) [reply inventory]: good idea! this was not in the initial PoC so can you raise an issue and add work on it ?

Comment (Miyurz) [reply inventory]: @RahulMahale  https://github.com/Miyurz/telemetry/issues/4

**Pull request merged**